website/docs/faqs/Snapshots/snapshot-target-is-not-a-snapshot-table.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Debug Snapshot target is not a snapshot table errors"
+description: "Debugging Snapshot target is not a snapshot table"
+sidebar_label: "Snapshot target is not a snapshot table"
+id: snapshot-target-is-not-a-snapshot-table
+---
+
+If you see the following error when trying to snapshot:
+
+> Snapshot target is not a snapshot table (missing `dbt_scd_id`, `dbt_valid_from`, `dbt_valid_to`)
+
+Double check that you haven't inadvertently caused your snapshot to behave like table materializations by setting its `materialized` config to be `table`. Prior to dbt version 1.4, it was possible to have a snapshot like this:
+
+```sql
+{% snapshot snappy %}
+  {{ config(materialized = 'table', ...) }}
+  ...
+{% endsnapshot %}
+```
+
+What would happen then is dbt treated snapshots like tables (issuing `create or replace table ...` statements) **silently** instead of actually snapshotting data (SCD2 via `insert` / `merge` statements). When upgrading to dbt versions 1.4 and above, dbt now raises a Parsing Error (instead of silently treating snapshots like tables) that reads:
```

Comment:
```suggestion
dbt is treating snapshots like tables (issuing `create or replace table ...` statements) **silently** instead of actually snapshotting data (SCD2 via `insert` / `merge` statements). When upgrading to dbt versions 1.4 and higher, dbt now raises a Parsing Error (instead of silently treating snapshots like tables) that reads:
```